src/quo2/components/numbered_keyboard/keyboard_key/style.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn get-label-color
+  [disabled? theme blur?]
+  (if disabled?
```

Comment:
consider flattening this structure with a `cond` 
```
(cond 
  (and disabled? (or (= :dark theme) blur?)) colors/white-opa-30
  (and disabled? (or (= :light theme) blur?)) colors/neutral-30
  (or (= :dark theme) blur?) colors/white
   :else  colors/neutral-100)
   ```

Reply:
Done